src/codecs/md/index.ts

```diff
@@ -607,6 +608,19 @@ function decodeCodeChunk(ext: Extension): stencila.CodeChunk {
       if (meta) codeChunk.meta = meta
       if (text) codeChunk.text = text
     }
+
+    if (nodes.length > 1) {
+      codeChunk.outputs = []
+      for (const outputContainer of nodes.slice(1)) {
+        if (!isA('Paragraph', outputContainer)) continue
+
+        outputContainer.content.forEach(output => {
+          if (codeChunk.outputs === undefined) return // TS being dumb
+          if (typeof output === 'string') codeChunk.outputs.push(output.trim())
+          else codeChunk.outputs.push(output)
+        })
+      }
+    }
   }
   return codeChunk
```

Comment:
While we are here, I suggest we refactor this to use factory functions:

```ts
const { programmingLanguage, meta, text = '' } = codeBlock
...
return stencila.codeChunk(text, { programmingLanguage, meta })
```

That compacts the object to exclude `undefined` properties, so then you don't need `if(programmingLanguage)` etc